### ISSUE TITLE
Combo counter improvements

### ISSUE THREAD

**Issue by OniKoneko** [JoelBesada/activate-power-mode#273]
I have a few suggestions for improving the look and feel of the combo counter, primarily adding a bit more customization for users. Justifications will follow each suggestion.

- Allow user controlled sizing of combo counter
  - This will help users that may use smaller windows from having text hidden under the counter
  - Possibly allow a user to set a max width for the number so larger numbers will resize to fit inside of that width
- Provide reset button for "max" combo counter
  - Let me see the highest I get each day!
- Have a number-only combo counter mode
  - Another space optimization, removes "combo" and the max combo counter.
  - Basically just the number
- Set "rainbow" threshold at which the combo number will cycle random colours
  - Another "fun" bit of feedback, lets higher combos feel extra special
  - Perhaps set threshold at which the counter shakes and spews particles
- Allow for multipliers
  - Let users specify things that can give them multipliers, such as closing parenthesis and using correct line endings for the language they're using.

**Comment by lexcast** [comment]
Hi @OniKoneko, thanks for your suggestions.

- Allow user controlled sizing of combo counter and Have a number-only combo counter mode
I think everything related to the look and feel of the combo widget should be handled by the user, 
with the command `aplication:open-your-stylesheet` you can insert custom css targeting whatever you want, for this the combo. You could use the dev tools to see how to target the combo.

- Provide reset button for "max" combo counter
Actually there is already a command for this: `activate-power-mode:reset-max-combo` for a button I would recommend the [tool-bar](https://github.com/suda/tool-bar) package.

- Allow for multipliers
I'm glad to say that the next version have multipliers :tada: every time you reach a new level the combo sets a multiplier.

- Set "rainbow" threshold at which the combo number will cycle random colours
Maybe related to the last one but not exactly the same, the combo changes its color when reach a new level. As I said is not the same.

**Comment by OniKoneko** [comment]
Awesome. I'm new to Atom so I didn't know about the CSS thing. I would like to argue (very politely) that it would be a very nice feature to just have that exposed in the package settings. :)

**Comment by sidpagariya** [comment]
Also, is there  chance we could reset the high score of the counter? Thanks!

**Comment by lexcast** [comment]
@OniKoneko There is already a setting for the combo counter opacity, so some few thing could be added if there is a PR for it. Maybe an option for font size would be enough.

**Comment by lexcast** [comment]
Hi @sidpagariya there is a command for it, `activate-power-mode:reset-max-combo`

**Comment by DolineanuMircea** [comment]
Hi @lexcast 
> Allow user controlled sizing of combo counter and Have a number-only combo counter mode
I think everything related to the look and feel of the combo widget should be handled by the user,
with the command aplication:open-your-stylesheet you can insert custom css targeting whatever you want, for this case the combo. You could use the dev tools to see how to target the combo

Could you specify the script for the style sheet that lets me do this, because I can't figure out how to access the class for the combo meter (I found the class itself). Sorry if this question is stupid, but my experience with css is zero, yet I love this pack and would like to know how to change the size.
